### PR TITLE
Add UML class diagram

### LINKED ISSUE #62: Milestone 2: Add a UML class diagram for ONLY the Models to the README

### PR DESCRIPTION
**Pull request by ajbalara**
# Description

Add UML files for the models and add a picture of the diagram to the README

## Issues

closes #62 

# How to test

Integration tests:
* Run `mvn test`

# Checklist
**I've added tests**
- [ ] Yes

- [x] No

This PR does not affect any tests

**I've updated the DB schema and UML diagram**
- [x] Yes
- [ ] No

### CODE REVIEW

**Review by JackieSL1 — COMMENTED** [review]
Do you think we need fields and methods in the diagram?

File: README.md [hunk 1]
```diff
@@ -105,6 +106,12 @@ The following diagram can be updated using the [Diagrams.net Integration Intelli
 
 ![DB schema](diagrams/db-schema.svg)
 
+## UML class Diagram for Models
+The following diagram can be updated by editing the corresponding drawio file in diagrams and then exporting the drawing
+to png.
+
+![UML class Diagram for Models](diagrams/ModelUML.drawio.png)
```
Comment (BraedenKloke) [inline README.md:113]: You should only need one file, not two - no exporting necessary. 
Collaborators updating the diagrams shouldn't have to export anything.

**Can you mimic how I have drawio implemented for the DB schema?**

**Comment by BraedenKloke** [comment]
> > Do you think we need fields and methods in the diagram?
> 
> Hm, @Sarvesh-Sabapathy @BraedenKloke @AliAbdollahian what do you guys think?

I think we need to add fields and methods. However, **I would suggest using the proper UML box notation but leaving the fields and methods boxes empty** until end of Milestone 2.

**Review by BraedenKloke — DISMISSED** [review]
# Changes requested
1. **Add boxes for fields and method:** Leave the boxes blank for now.
2. **Consolidate diagram into a single file:** Collaborators shouldn't need to worry about exporting a file with each change.

**Comment by AliAbdollahian** [comment]
Its always a good practice to clarify the behaviour and data of the system

**Comment by ajbalara** [comment]
> # Changes requested
> 1. **Add boxes for fields and method:** Leave the boxes blank for now.
> 2. **Consolidate diagram into a single file:** Collaborators shouldn't need to worry about exporting a file with each change.

Implemented requested changes in most recent commit to branch

**Review by JackieSL1 — COMMENTED** [review]
~~Also, should the aggregation arrows from `Question` to `Response` and `Question` to `Survey` be facing the other direction?~~

Nevermind!

**Review by JackieSL1 — APPROVED** [review]

**Pull request merged**